ci/build-push-multiplatform.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+set -exuo pipefail
+
+docker buildx build --push --platform=linux/amd64     --build-arg FLUENTD_FLAVOR=debian       --build-arg BUILD_TAG="${BUILD_TAG}" --tag "${REPO_URL}:${BUILD_TAG}-amd64" . & amd64_pid=$!
+docker buildx build --push --platform=linux/arm/v7    --build-arg FLUENTD_FLAVOR=debian-armhf --build-arg BUILD_TAG="${BUILD_TAG}" --tag "${REPO_URL}:${BUILD_TAG}-arm"   . & arm_pid=$!
+docker buildx build --push --platform=linux/arm64/v8  --build-arg FLUENTD_FLAVOR=debian-arm64 --build-arg BUILD_TAG="${BUILD_TAG}" --tag "${REPO_URL}:${BUILD_TAG}-arm64" . & arm64_pid=$!
+
+wait $amd64_pid
+wait $arm_pid
+wait $arm64_pid
+
+docker manifest create --amend \
+    "${REPO_URL}:${BUILD_TAG}" \
+    "${REPO_URL}:${BUILD_TAG}-amd64" \
+    "${REPO_URL}:${BUILD_TAG}-arm" \
+    "${REPO_URL}:${BUILD_TAG}-arm64"
+
+docker manifest push "${REPO_URL}:${BUILD_TAG}"
```

Comment:
Why is this different from https://github.com/SumoLogic/sumologic-kubernetes-setup/blob/a1e3db6c2ee4684eaf8980a128a700950b1e9162/Makefile#L22-L27?

Reply:
We cannot use a single `docker buildx build` command, because we need to use different base Fluentd image tag for each architecture. We modify the tag with a build arg, so we need to execute three different `docker buildx build` commands with different values for the `FLUENTD_ARCH` build arg, and then put them together into one tag with the `docker manifest create` command.